spec/Powertrain/Battery.vspec

```diff
@@ -83,3 +84,103 @@
   unit: kWh
   description: The accumulated energy leaving HV battery for propulsion and auxiliary loads over lifetime.
 
+
+- Charging:
+  type: branch
+  description: Properties related to battery charging
+  
+- Charging.ChargeLimit:
+  datatype: UInt8
+  type: actuator
+  min: 0
+  max: 100
+  unit: percent
+  description: Maximum charge level for battery, can potentially be set manually.
```

Comment:
Ampere? Prozent? Vielleicht besser in description
"Stop charging at x % of net capacity" oder sowas